index.html

```diff
@@ -3,9 +3,67 @@
   <head>
     <meta charset="UTF-8" />
     <title>Tribute Page</title>
-    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="css/style.css" />
   </head>
 
-  
+  <body>
+    <main id="main">
+      <div class="header">
+        <h1 id="title">Dr. Norman Borlaug</h1>
+        <p>The man who saved a billion lives</p>
+      </div>
```

Comment:
Consider using semantic elements.

Reply:
- [ ] The comment here is not very clear and easy to understand. please kindly provide more information on the suggested changes so that the code author will understand better💪 and include a reason "why" for each requested change. 

Example
Let us use a `<header>` tag instead of a `<div>` and then we can provide a reason . . .

**_[OPTIONAL]_** It will be nice if you use a couple of emojis 🤗 and/or gifs in the review. they make the code review more human-friendly 👽🤖.

Reply:
Consider using semantic elements, in this case, you can choose a <nav> or <header> tag instead a <div>, this practice has many advantages for example making code easier for people to navigate when updating features or fixing bugs. 👏